### ISSUE TITLE
[Bug]: 0.8.0 - MQTT connection error?

### ISSUE THREAD

**Issue by SSpecken** [bropat/ioBroker.eufy-security#219]
### Adapter version

0.8.0

### JS-Controller version

3.3.22

### Node version

12.22.7

### Operating System type

Linux

### Operating system version

Raspbian GNU/Linux 10 (buster)

### Describe the bug

After updating to update to 0.8.0 I'm getting these errors every second:

```
eufy-security.0 | 2022-02-05 16:20:36.811 | error | MQTT Error [{"code":5}]
eufy-security.0 | 2022-02-05 16:20:35.673 | info | MQTT connection closed
```

But I don't find any information about enabling/disabling/configuring MQTT. Only that this version is supporting the MQTT interface of the T8520 Smart Lock now.

Do you have any hints how to solve this?

I've got these devices:
1 x HomeBase T8010
1 x Doorbell T8210
1 x Motion Sensor T8910
2 x IndoorCam Pan 2K T8410

### To reproduce

1. start eufy-security instance
2. wait some seconds
3. see errors in the protocol

**Comment by Baenker** [comment]
Habe eben die Version installiert bei mir läuft das log voll mit der Meldung immer abwechselnd in Dauerschleife:

`

eufy-security.0 | 2022-02-05 20:02:57.937 | info | MQTT connection closed
-- | -- | -- | --
eufy-security.0 | 2022-02-05 20:02:57.936 | error | MQTT Error [{"code":5}]

`

**Comment by bropat** [comment]
Sorry guys. Fixed in version 0.8.1
